### PR TITLE
Update step_5.md

### PR DESCRIPTION
**Pull request by MartinGallauner**
improves wording and punctuation.

### CODE REVIEW

File: website/docs/pact_nirvana/step_5.md [hunk 1]
```diff
@@ -37,34 +37,34 @@ By using branches alongside application versions in the broker allows you to sep
...
 
-To keep a green build, in your provider’s CI, we will utilise consumer version selectors to verify the main branch, rather than verifying the latest overall pact.
+To maintain a green build in your provider’s CI, we will use consumer version selectors to verify the main branch rather than the latest overall pact.
 
-If you use feature branches for your consumer development, it is recommended to publish the pact with pacticipant version and the name of the branch. If you use feature toggles, the branch could be the name of the feature toggle. Your Pact client library will allow you to configure the name of the branch to be applied when you publish your pacts.
+If you use feature branches for your consumer development, it is recommended to publish the pact with the participant version and the name of the branch. If you use feature toggles, the branch could be the name of the feature toggle. Your Pact client library will allow you to configure the name of the branch to be applied when you publish your pacts.
```
Comment (YOU54F) [inline website/docs/pact_nirvana/step_5.md:42]: I think this should possibly remain as `pacticipant`

and possibly link the terminology section for the word here, if you feel the reader may be confused here

https://docs.pact.io/getting_started/terminology#pacticipant

wdyt?

Comment (MartinGallauner) [reply website/docs/pact_nirvana/step_5.md:42]: Alright, now I understand where that is coming from.
When I read that the first time, I thought it was a typo, and someone was typing "pact" just a few times too often before that.
I'm okay with both options but prefer the word **participant** as it has the same semantics but is an actual word

Comment (MartinGallauner) [reply website/docs/pact_nirvana/step_5.md]: I just changed it back to pacticipant and linked the definition.
Thansk for the review!

Comment (YOU54F) [reply website/docs/pact_nirvana/step_5.md]: >When I read that the first time, I thought it was a typo, and someone was typing "pact" just a few times too often before that.

It happens to us all! Thanks, the link to the definition will certainly help.

Maybe setting some of the terminology at the start of guide might be helpful.

Appreciate the edits!

**Pull request merged**